src/test/ui/generic/generic-extern-lifetime.stderr

```diff
@@ -9,12 +9,24 @@ error[E0261]: use of undeclared lifetime name `'a`
    |
 LL |    pub fn life4<'b>(x: for<'c> fn(&'a i32));
    |                                    ^^ undeclared lifetime
+   |
+   = note: for more information on higher-ranked polymorphism, visit https://doc.rust-lang.org/nomicon/hrtb.html
```

Comment:
btw, can this note be moved to after the `help:`? The blog post analogy to this would be having the "read more" link before the introduction.

Reply:
This is part of a separate bug where all suggestions come always at the end. I would rather tackle that in a separate PR, ideally after fully migrating to annotate_snippets.